### PR TITLE
Fixing SqlServerOperations.java

### PR DESCRIPTION
**Pull request by masonwheeler**
Fixing `SqlServerOperations` to match the changes to the `SqlOperations` interface that happened while this was in development.

### CODE REVIEW

File: airbyte-integrations/connectors/destination-mssql/src/main/java/io/airbyte/integrations/destination/mssql/SqlServerOperations.java [hunk 1]
```diff
@@ -30,16 +30,14 @@
...
-import java.util.stream.Stream;
 
 public class SqlServerOperations implements SqlOperations {
 
   @Override
   public void createSchemaIfNotExists(JdbcDatabase database, String schemaName) throws Exception {
     final String query = String.format("IF NOT EXISTS ( SELECT * FROM sys.schemas WHERE name = '%s') EXEC('CREATE SCHEMA [%s]')",
-        schemaName,
-        schemaName);
+            schemaName,
```
Comment (davinchia) [inline airbyte-integrations/connectors/destination-mssql/src/main/java/io/airbyte/integrations/destination/mssql/SqlServerOperations.java:39]: is this from formatting?

**Review by davinchia — APPROVED** [review]
Looks good!

I would run the mssql integration test `/test connector=destination-mssql` before merging just to be sure.

File: airbyte-integrations/connectors/destination-mssql/src/main/java/io/airbyte/integrations/destination/mssql/SqlServerOperations.java (hunk 1 above)
Comment (masonwheeler) [reply airbyte-integrations/connectors/destination-mssql/src/main/java/io/airbyte/integrations/destination/mssql/SqlServerOperations.java:39]: Yeah, the formatter did a bunch of stuff to it.  The substantive changes are mostly down at the bottom.

**Comment by masonwheeler** [comment]
Fixing some issues with `SqlServerOperations`, which was out of sync with recent changes to `SqlOperations`.

**Pull request merged**